packages/ckeditor5-dev-release-tools/lib/tasks/updatedependencies.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const fs = require( 'fs-extra' );
+const { globSync } = require( 'glob' );
+const { logger } = require( '@ckeditor/ckeditor5-dev-utils' );
+
+/**
+ * The purpose of this script is to update all eligible dependencies to a version specified in the `options.version`. The following packages
+ * are taken into consideration:
+ *
+ * - The root package located in `options.cwd` path.
+ * - All packages located in the `options.packagesDirectory` path relative to `options.cwd`.
+ *
+ * The eligible dependencies are distinguished by the return value from the `options.shouldUpdateVersionCallback` function. Only if this
+ * callback returns a truthy value for a given dependency, its version will be updated.
+ *
+ * @param {Object} options
+ * @param {String} options.version Target version to which all of the eligible dependencies will be updated.
+ * @param {String} options.shouldUpdateVersionCallback Callback function that decides whether to update a version for a dependency.
```

Comment:
I know that we don't wan't to leak any private packages names, but I have a feeling that this callback should be optional.

Reply:
Assuming so, how would you like to determine if a package should be updated? I do not see any other solution that allows us to use tooling outside the CKEditor 5 project. 

E.g., the package generator. We do not want to update CKEditor 5 feature packages while releasing the tool. We only want to update tools and the generator itself.

Reply:
We talked about it and it's not as easy as I imagined. So not using default value is okay for me :)